CRM/Dedupe/Merger.php

```diff
@@ -586,36 +586,23 @@ public static function retrieveFields($main, $other) {
    *                              A 'safe' value skips the merge if there are any un-resolved conflicts.
    *                              Does a force merge otherwise.
```

Comment:
Good point - I just amended it (I did it straight through the web UI so cross fingers it doesn't need a style fix now!)